src/db.py

```diff
@@ -1,5 +1,11 @@
+
+import sqlite3
 from dataclasses import dataclass
 
+
+# queries to create the db tables
+
+
```

Comment:
these changes no longer makes sense due to #5

Reply:
removed